### PR TITLE
Change Agent namespace, not HOSTED_CONTROL_PLANE_NAMESPACE

### PR DESCRIPTION
**Pull request by LiangquanLi930**
Add AGENT_NAMESPACE configuration to allow the workflow to use namespaces not intended for CP.

### CODE REVIEW

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.16-periodics-e2e-mce-agent-connected-ovn-ipv4-metal3-conformance

File: ci-operator/step-registry/hypershift/agent/create/add-worker-metal3/hypershift-agent-create-add-worker-metal3-commands.sh [hunk 1]
```diff
@@ -10,28 +10,36 @@ source "${SHARED_DIR}/packet-conf.sh"
...
 metadata:
   name: ${CLUSTER_NAME}
-  namespace: ${HOSTED_CONTROL_PLANE_NAMESPACE}
+  namespace: ${AGENT_NAMESPACE}
 spec:
   cpuArchitecture: x86_64
   pullSecretRef:
     name: pull-secret
-  sshAuthorizedKey: ${SSH_PUB_KEY}
+  sshAuthorizedKey: '${SSH_PUB_KEY}'
```
Comment (jparrill) [inline ci-operator/step-registry/hypershift/agent/create/add-worker-metal3/hypershift-agent-create-add-worker-metal3-commands.sh:42]: Double quotes?

Comment (LiangquanLi930) [reply ci-operator/step-registry/hypershift/agent/create/add-worker-metal3/hypershift-agent-create-add-worker-metal3-commands.sh]: Thank you for your reminder, updated it

**Pull request merged**